Yosemite/Yosemite/Actions/ShipmentAction.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import Networking
+
+
+// MARK: - ShipmentAction: Defines all of the Actions supported by the ShipmentStore.
+//
+public enum ShipmentAction: Action {
+
+    /// Synchronizes the all of the shipment tracking associated with the provided `siteID` and `orderID`
```

Comment:
Non-native speaker here, but should this read as `Synchronizes all the shipment...`?

Reply:
Yep! I was typing comments too quickly I guess 😄